internal/shim-sgx/Cargo.toml

```diff
@@ -28,8 +28,8 @@ noted = "^1.0.0"
 xsave = { version = "2.0.2" }
 rcrt1 = "1.0.0"
 lset = { git = "https://github.com/enarx/lset", rev = "5c2feed24d85442725f55d812315854e1b29595d"  }
-sgx = "0.3"
-mmledger = { git = "https://github.com/enarx/mmledger", rev = "843bd70e5266bcff912d651af0a9c07a413b5626" }
+sgx = { git = "https://github.com/enarx/sgx", rev = "f6de78316cdd74abf825dc3f6794f6f43f744e85" }
+mmledger = { git = "https://github.com/jarkkojs/mmledger", rev = "373a6d45bfc85cf6c026657cc952888a83a7f084" }
```

Comment:
I will approve, once this points to upstream

Reply:
awesome! ping @npmccallum

Reply:
Test pass without mmledger changes so I took it away. @npmccallum please anyway merge it because I need to continue from that point for mremap(). It's a more stable baseline.